TourMate/TourMate/Frontend/Plan/Views/PlanFormView/PlanFormViewModel.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  PlanFormViewModel.swift
+//  TourMate
+//
+//  Created by Terence Ho on 1/4/22.
+//
+
+import Foundation
+import Combine
+
+@MainActor
+class PlanFormViewModel: ObservableObject {
+    let lowerBoundDate: Date
```

Comment:
I have decoupled the rest, now only AddPlanViewModel takes in a trip, which I think makes sense because to add a plan we need to know properties in a trip.